src/components/CtaCard/CtaCard.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import "./CtaCard.css";
+
+const CtaCard = ({ children, title, iconHandle, link }) => {
+  return (
+    <div className="CtaCard">
+      <a href={link} title={title}>
```

Comment:
Errr... Any way of passing it as a React element? such as `<Link to="/internal/gatsby/link">Text</Link>` or `<a href="http://outside.internet.example.com">Text</a>`? We'd like to keep gatsby prefetching functionality by unsing the internal `Link` if possible